src/git-extract

```diff
@@ -54,17 +54,18 @@ function preconditions {
 }
 
 
-function steps {
-  echo_if_true "stash_open_changes" "$INITIAL_OPEN_CHANGES"
+function should_stash_open_changes {
```

Comment:
Also, could this become a variable that the framework sets to `true`, and which the script can set to `false`? This would make this script more concise.